src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/PersonRepositoryCustomImplIntTest.kt

```diff
@@ -33,25 +29,11 @@ class PersonRepositoryCustomImplIntTest : IntegrationTestBase() {
     assertThat(results[0].personId).isEqualTo(UUID.fromString("ddf11834-e2a3-11ed-b5ea-0242ac120002"))
   }
 
-  @Test
-  fun `should return person record for known provided middle names`() {
-    // Given
-    val searchRequest = PersonSearchRequest(
-      surname = "Mahoney",
-      forenameThree = "bob",
-    )
-
-    // When
-    val results = personRepository.searchByRequestParameters(searchRequest)
-
-    // Then
-    assertThat(results).hasSize(1)
-    assertThat(results[0].personId).isEqualTo(UUID.fromString("eed4a9a4-d853-11ed-afa1-0242ac120002"))
-  }
```

Comment:
creation of a person with middle names is unsupported at the moment

Reply:
The ticket I just picked up mentioned this so better keep it.

Reply:
Is the functionality well-defined? It does not work at the moment, in the sense that any data created by our app will only have first name (given name) and family name, so this search can't succeed. Moreover, the existing `from` method in PersonSearchRequest does not allow you to create a search on middle name, and that is what is currently used in production. 

So this test will only work on data inserted directly into the database - something which does not happen when consuming messages.

By all means we can add the test back in once it is testing something which is in use in production code - test driving that implementation perhaps.